ggshield/core/scan/scan_mode.py

```diff
@@ -15,4 +15,5 @@ class ScanMode(Enum):
     IAC_DIRECTORY = "directory"
     IAC_DIFF = "diff"
     DOCSET = "docset"
-    SCA_DIRECTORY = "sca_directory"
+    SCA_DIRECTORY = "directory"
+    SCA_DIFF = "diff"
```

Comment:
As we use the same `PRE_COMMIT`, `CI` etc as secret, could we also have just a generic `DIRECTORY` and generic `DIFF` that are shared with IaC ?